config/bin/src/dummy.sh

```diff
@@ -19,7 +19,10 @@ function dummy_content_modules {
 
 ##
 # Execute dummy migration.
+# 'dummy_content_migration' is set to 1 during the process, 0 when done.
 ##
 function dummy_content_migrate {
+  drupal_drush vset --exact dummy_content_migration 1
```

Comment:
@anvmn 
I think it will be better to use `drupal_is_cli()` in order to check this ...